src/main/java/org/eln2/mc/common/CommonEvents.kt

```diff
@@ -38,4 +50,53 @@ object CommonEvents {
             }
         }
     }
+
+    @SubscribeEvent
+    fun onEntityJoinedWorld(event: EntityJoinWorldEvent) {
+        //Warn new users that we collect analytics
+        if (event.world.isClientSide && event.entity is Player) {
+            if (Eln2.config.enableAnalytics)  {
+                val acknowledgementFile = FMLPaths.CONFIGDIR.get().resolve("ElectricalAge/acknowledgements.yml").toFile()
+                if (!acknowledgementFile.exists()) {
+                    try {
+                        acknowledgementFile.parentFile.mkdirs()
+                        acknowledgementFile.createNewFile()
+                        acknowledgementFile.writeText(Yaml.default.encodeToString(AnalyticsAcknowledgementsData.serializer(), AnalyticsAcknowledgementsData(mutableMapOf())))
+                    } catch (ex: Exception) {
+                        when (ex) {
+                            is IOException, is SecurityException -> {
+                                Eln2.LOGGER.warn("Unable to write default analytics acknowledgements config")
+                                Eln2.LOGGER.warn(ex.localizedMessage)
+                            }
+                            else -> throw ex
+                        }
+                    }
+                }
+                val uuid: String = event.entity.uuid.toString()
+                val acknowledgements: AnalyticsAcknowledgementsData = try {
+                    Yaml.default.decodeFromStream(AnalyticsAcknowledgementsData.serializer(), acknowledgementFile.inputStream())
+                } catch (ex: IOException) {
+                    AnalyticsAcknowledgementsData(mutableMapOf())
+                }
+                if (acknowledgements.entries.none { (k, _) -> k == uuid } || acknowledgements.entries[uuid]!! < (System.currentTimeMillis()) - THIRTY_DAYS_AS_MILLISECONDS) {
+                    event.entity.sendMessage(TranslatableComponent("misc.eln2.acknowledge_analytics").withStyle(ChatFormatting.RED), Util.NIL_UUID)
```

Comment:
Does it need to be red? I think just standard text color is sufficient.. what do other mods do for startup messages? I feel like red should be reserved for things like crashes or critical errors.